### PR TITLE
New greenish theme

### PR DESCRIPTION
**Pull request by harpreet9629**
Here's New Theme for BitClout

Screenshot_5

### CODE REVIEW

**Review by lazynina — CHANGES_REQUESTED** [review]
There are three different theme files added. Which one is the one you're adding? Also, it needs to be imported in app-module; otherwise, it can't be selected in the settings page.

Also, please remove the addition of button properties as that is a separate feature.

**Comment by harpreet9629** [comment]
Ok i do, thank you

**Comment by harpreet9629** [comment]
@lazynina Its Fixed 👍

File: src/app/theme/greenish-theme.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "--hover": "#0294A5",
+    "--border": "#00515A",
+    "--mborder": "#00515A",
+    "--filter": "invert(98%) sepia(1%) saturate(264%) hue-rotate(181deg) brightness(116%) contrast(100%)",
+    "--unread": "#034753",
+    "--topbar": "#034753",
+    "--cblue": "#238eff",
+    "--cred": "#e0245e",
+    "--cgreen": "#17BF63",
+    "--loading": "#ffffff",
```
Comment (lazynina) [inline src/app/theme/greenish-theme.ts:25]: add button property
```suggestion"
    "--button": "#03A8BB",
    "--loading": "#ffffff", 
```

File: src/app/theme/greenish-theme.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "--filter": "invert(98%) sepia(1%) saturate(264%) hue-rotate(181deg) brightness(116%) contrast(100%)",
+    "--unread": "#034753",
+    "--topbar": "#034753",
+    "--cblue": "#238eff",
+    "--cred": "#e0245e",
+    "--cgreen": "#17BF63",
+    "--loading": "#ffffff", 
+  },
+};
+//"--button": "#03A8BB",
```
Comment (lazynina) [inline src/app/theme/greenish-theme.ts:28]:
```suggestion
```

**Review by lazynina — CHANGES_REQUESTED** [review]

File: src/app/theme/gray-theme.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { Theme } from "./symbols";
```
[the remark this answers is not in the document]
Comment (harpreet9629) [reply src/app/theme/gray-theme.ts]: It's Deleted

File: src/app/theme/greenish-theme.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { Theme } from "./symbols";
```
[the remark this answers is not in the document]
Comment (harpreet9629) [reply src/app/theme/greenish-theme.ts:1]: Its Done i changed theme name in greenish-theme.ts file
image

File: src/app/theme/icy-dark-theme.ts [hunk 5]
```diff
@@ -22,6 +22,7 @@ export const icydarkTheme: Theme = {
     "--cblue": "#238eff",
     "--cred": "#e0245e",
     "--cgreen": "#17BF63",
+    "--button": "#3a9eea",
```
[the remark this answers is not in the document]
Comment (harpreet9629) [reply src/app/theme/icy-dark-theme.ts:25]: It show me my branch already merge with main

**Review by lazynina — APPROVED** [review]

**Comment by harpreet9629** [comment]
Thank You Guys :)

I will come back with more changes

**Comment by LvlUpGuy** [comment]
This almost has a "Emerald" feeling to it.

I like it and will most assuredly try it.

**Pull request merged**